### PR TITLE
feat: placeholder landing page for payments

### PR DESCRIPTION
**Pull request by LinHuiqing**
## Problem

No landing page for payments - more difficult to onboard admins and convince them to use us for payment flows.

Closes FRM-986

## Solution

**Breaking Changes** 

- [x] No - this PR is backwards compatible  

**Features**:

- Placeholder landing page for payments

**Improvements**:

- Create inverseOutline button variant

## Screenshots

Desktop:

screencapture-localhost-3000-payments-2023-06-08-15_14_08

Mobile:

screencapture-localhost-3000-payments-2023-06-08-15_14_41

## Tests

- [ ] Check that landing page is as expected on desktop ([design](https://www.figma.com/file/CMq0CJtawn7NGM7Zdpi8Ti/Form-Design-Master?type=design&node-id=17464-183101&t=dHH857IVxtAbRsYj-0)).
- [ ] Check that landing page is as expected on mobile.

To check button theme works on banner buttons as well:
- [ ] Check that the "Learn more" button in the payments banner on main landing page is okay.
- [ ] Check that the "Learn more" button in the payments banner on login page is okay.

### CODE REVIEW

File: frontend/src/pages/LandingPayments/components/FeatureGridItem.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import { SectionBodyText } from './SectionBodyText'
+
+interface FeatureGridItemProps {
+  image?: string
+  title: string
+  description: string
+}
+
+export const FeatureGridItem = ({
```
Comment (KenLSM) [inline frontend/src/pages/LandingPayments/components/FeatureGridItem.tsx]: Great abstraction!

**Review by KenLSM — APPROVED** [review]
LGTM.

I think FRM-972 is a duplicate of this ticket.

File: frontend/src/pages/LandingPayments/components/FeatureGridItem.tsx (hunk 1 above)
Comment (LinHuiqing) [reply frontend/src/pages/LandingPayments/components/FeatureGridItem.tsx]: Nono this one I just copied over from `frontend/src/pages/Landing/components/FeatureGridItem.tsx` actually, maybe we should just shift those components out so we can reuse, or i just import from there oop

**Comment by LinHuiqing** [comment]
> I think [FRM-972](https://linear.app/ogp/issue/FRM-972) is a duplicate of this ticket.

also @KenLSM, its not a duplicate because the actual landing page should have the email input field and the stats for our payments feature

File: frontend/src/pages/LandingPayments/components/FeatureGridItem.tsx (hunk 1 above)
Comment (LinHuiqing) [reply frontend/src/pages/LandingPayments/components/FeatureGridItem.tsx]: refactored to reuse components here: [06a061d](https://github.com/opengovsg/FormSG/pull/6440/commits/06a061d7051005b5bbc26fdc5360d1a250dc3672)

**Pull request merged**